README.md

```diff
@@ -50,6 +61,12 @@ Certificate password. Default `""`.
 
 Output path of ipa. Default `"output.ipa"`.
 
+### `append-github-sha`
+
+Append GITHUB_SHA to output filename.
+
+*example: if `outpuh-path=app.ipa` then the resulting filename would be `app-{github-sha}.ipa`*
+
```

Comment:
You can easily do the same thing with `append-github-sha` by using `github.sha` in Context.

```
- uses: yukiarrr/ios-build-action@v1.0.0
  with:
    project-path: Unity-iPhone.xcodeproj
    p12-base64: ${{ secrets.P12_BASE64 }}
    mobileprovision-base64: ${{ secrets.MOBILEPROVISION_BASE64 }}
    code-signing-identity: ${{ secrets.CODE_SIGNING_IDENTITY }}
    team-id: ${{ secrets.TEAM_ID }}
    output-path: app-${{ github-sha }}.ipa
```

So, isn't `append-github-sha` unnecessary?

Reply:
We wanted to make this optional, that's why we added a new parameter, because maybe you don't want the SHA appended in your artifacf.

Reply:
How about removing `append-github-sha`?

If add `append-github-sha`, for example if want to append new info to the filename other than SHA, you need to add a new parameter again. (`append-github-XXX`)

So, if want to append info such as SHA to the filename, I think it's better to use `github.sha` on the user side of `ios-build-action`. (`output-path: app-${{ github.sha }}.ipa`)

Reply:
Makes sense to me. I'm going to remove it.